From e7b3f2a4f144ac1c756a7c36242b698f905b9c7d Mon Sep 17 00:00:00 2001
From: Michele Locati <michele@locati.it>
Date: Thu, 29 Nov 2018 16:44:12 +0100
Subject: [PATCH] Test2

---
 src/JShrink/Minifier.php | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/JShrink/Minifier.php b/src/JShrink/Minifier.php
index 86734da..fe4b519 100644
--- a/src/JShrink/Minifier.php
+++ b/src/JShrink/Minifier.php
@@ -181,7 +181,7 @@ class Minifier
                 // new lines
                 case "\n":
                     // if the next line is something that can't stand alone preserve the newline
-                    if (strpos('(-+{[@', $this->b) !== false) {
+                    if ($this->b !== false && strpos('(-+{[@', $this->b) !== false) {
                         echo $this->a;
                         $this->saveString();
                         break;
@@ -224,7 +224,7 @@ class Minifier
                             // check for some regex that breaks stuff
                             if ($this->a === '/' && ($this->b === '\'' || $this->b === '"')) {
                                 $this->saveRegex();
-                                continue;
+                                continue 3;
                             }
 
                             echo $this->a;
-- 
2.19.2.windows.1

